dependent/jospt.csl

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <info>
+    <title>JOSPT (Journal of Orthopaedic and Sports Physical Therapy)</title>
```

Comment:
Oh, and to write an ampersand, use `&amp;` (ampersands need to be 'escaped' in HTML)